lib/PuppeteerSharp.Tests/EvaluationTests/PageEvaluateTests.cs

```diff
@@ -323,7 +323,7 @@ public async Task ShouldProperlySerializeNullFields()
         public async Task ShouldAcceptObjectHandleAsAnArgument()
         {
             await Page.SetContentAsync("<section>42</section>");
-            var element = await Page.QuerySelectorAsync("section");
+            object element = await Page.QuerySelectorAsync("section");
```

Comment:
We don't need this. I can merge this PR if it's only about the gitignore.

Reply:
Ok
Anyway what was that test even about? It sounds straightforward but I dont quiet get what it's testing. Should it be removed?